backend/src/apiserver/client_manager.go

```diff
@@ -136,7 +136,7 @@ func (c *ClientManager) UUID() util.UUIDGeneratorInterface {
 	return c.uuid
 }
 
-func (c *ClientManager) init() {
+func (c *ClientManager) init(clientQPS float32, clientBurst int) {
```

Comment:
Following initialization uses `common.GetXXX()` to retrieve configuration. Can you take a look if you can migrate to this consistent way?